app/components/wallet/trezorConnect/common/ErrorBlock.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { intlShape } from 'react-intl';
+
+import LocalizableError from '../../../../i18n/LocalizableError';
+import type { ProgressInfo } from '../../../../stores/ada/TrezorConnectStore';
+import styles from './ErrorBlock.scss';
+
+type Props = {
+  progressInfo: ProgressInfo,
+  error: ?LocalizableError,
+};
+
+@observer
+export default class ErrorBlock extends Component<Props> {
+
+  static contextTypes = {
+    intl: intlShape.isRequired
+  };
+
+  render() {
+    const { intl } = this.context;
+    let errorText = '';
+    try {
+      errorText = (this.props.error) ? intl.formatMessage(this.props.error) : '';
+    } finally {} // eslint-disable-line no-empty
```

Comment:
this seems odd to have an empty `finally` and no `catch`. I would delete the `finally` and add a catch that w/ Logger

Reply:
DONE